File: src/litagreining.lua
r, g, b = io.read("*n", "*n", "*n")
if r > g and r > b then
    print('raudur')
elseif g > r and g > b then
    print('graenn')
elseif b > r and b > g then
    print('blar')
elseif r == g and b < g then
    print('gulur')
elseif r == b and g < b then
    print('fjolubleikur')
elseif b == g and r < b then
    print('blagraenn')
elseif r == g and g == b and b == 0 then
    print('svartur')
elseif r == g and g == b and b == 255 then
    print('hvitur')
elseif r == g and g == b then
    print('grar')
else
    print('othekkt')
end

File: src/litagreining.py
r, g, b = [int(d) for d in input().split()]
if r > g and r > b:
    print("raudur")
elif g > r and g > b:
    print("graenn")
elif b > r and b > g:
    print("blar")
elif r == g and b < g:
    print("gulur")
elif r == b and g < b:
    print("fjolubleikur")
elif b == g and r < b:
    print("blagraenn")
elif r == g and g == b and b == 0:
    print("svartur")
elif r == g and g == b and b == 255:
    print("hvitur")
elif r == g and g == b:
    print("grar")
else:
    print("othekkt")
